### PR TITLE
Intercoms

### PR DESCRIPTION
**Pull request by Terraspark4941**
## About the PR

Ports the intercom sprite, and adds a bunch of prefabricated intercoms for the colony,  each department and each marines squad. 

## Why / Balance

#838

## Media

image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by Terraspark4941** [comment]
do forgive me if something is wrong lore/function-wise, this is my 3rd/4th time dabbling in CM/TGMC lol

File: Resources/Prototypes/_CM14/Entities/Structures/Wallmounts/intercom.yml [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    - MarineFoxtrot
+
+- type: entity
+  id: CMIntercomColony
+  parent: CMIntercom
+  suffix: Colony
+  components:
+  - type: Intercom
+    supportedChannels:
+    - MarineCommon
```
Comment (DangerRevolution) [inline Resources/Prototypes/_CM14/Entities/Structures/Wallmounts/intercom.yml:239]:
```suggestion
```

**Review by DangerRevolution — APPROVED** [review]
Intercoms on the ground shouldn't get access to the Alamyer's channel

**Comment by DangerRevolution** [comment]
Test fail doesn't *seem* to be real

File: Resources/Prototypes/_CM14/Entities/Structures/Wallmounts/intercom.yml (hunk 1 above)
Comment (Terraspark4941) [reply Resources/Prototypes/_CM14/Entities/Structures/Wallmounts/intercom.yml:239]: i see

**Comment by Terraspark4941** [comment]
oh cool just realized redoing the test made it pass
weird but expected

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**